### PR TITLE
[CE-2605] generic request example

### PR DESCRIPTION
**Pull request by akedalat**

### CODE REVIEW

File: generic-request-example/src/#generic_request/handler.test.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { OperationHandlerTestSetup } from '@trayio/cdk-dsl/connector/operation/OperationHandlerTest';
+import { OperationHandlerResult } from '@trayio/cdk-dsl/connector/operation/OperationHandler';
+import { genericRequestHandler } from './handler';
+import '@trayio/cdk-runtime/connector/operation/OperationHandlerTestRunner';
+import 'dotenv/config'
```
Comment (gnyani-trayio) [inline generic-request-example/src/#generic_request/handler.test.ts:5]: why do you need a dotenv here?

File: generic-request-example/src/#generic_request/handler.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+							client_secret: input.auth_form_input.client_secret,
+							account_id: input.auth_form_input.account_id,
+							audience: 'https://api.bizzabo.com/api',
+							grant_type: 'client_credentials',
+						})
+				})
+				.handleResponse((ctx, input, response) => {
+					return response.parseWithBodyAsJson<GetAccessTokenRes>((body) => {
+						return OperationHandlerResult.success({
+							// You must respond with status_code, headers, and body
```
Comment (gnyani-trayio) [inline generic-request-example/src/#generic_request/handler.ts:32]: why is this a must ?

File: generic-request-example/tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (gnyani-trayio) [inline generic-request-example/tsconfig.json:1]: may be fix the formatting here?

File: generic-request-example/src/#generic_request/handler.test.ts (hunk 1 above)
Comment (akedalat) [reply generic-request-example/src/#generic_request/handler.test.ts:5]: oops forgot to remove

File: generic-request-example/src/#generic_request/handler.ts (hunk 2 above)
Comment (akedalat) [reply generic-request-example/src/#generic_request/handler.ts:32]: API service expect this otherwise it throws error

**Review by gnyani-trayio — APPROVED** [review]

**Pull request merged**